app/javascript/lib/metrics-api.js

```diff
@@ -422,7 +423,15 @@ export function log(name, props = {}) {
   init = Object.assign(init, body)
 
   if ('SCP' in window || metricsApiMock) {
-    fetch(`${bardDomain}/api/event`, init)
+    fetch(`${bardDomain}/api/event/`, init).then(response => {
+      // log failed attempts to connect with Bard to Sentry
+      if (!response.ok) {
+        logJSFetchExceptionToSentry(response, 'failed to log event to Bard')
+      }
+    // log errored attempts to connect with Bard to Sentry
+    }).catch(error => {
+      logJSFetchErrorToSentry(error, 'failed to log event to Bard')
```

Comment:
Non-blocking: It might be better to have a different log message here, like `error in logging event to Bard` to denote that the request threw an error, rather than a complete round-trip request that had a non-OK status code.

Reply:
oh good call - i'll update that (and the one below)